packages/dev-tools/server/graphql/GraphQLSchema.ts

```diff
@@ -576,7 +576,22 @@ const resolvers = {
     },
     async setProjectSettings(parent, { settings }, context) {
       const currentProject = context.getCurrentProject();
+      let previousSettings = await ProjectSettings.readAsync(currentProject.projectDir);
       let updatedSettings = await ProjectSettings.setAsync(currentProject.projectDir, settings);
+
+      // If 'tunnel' wasn't previously configured and it will be as a result of this request, start tunnels.
+      if (previousSettings.hostType !== 'tunnel' && updatedSettings.hostType === 'tunnel') {
+        try {
+          await Project.startTunnelsAsync(currentProject.projectDir);
+        } catch (e) {
+          ProjectUtils.logDebug(
```

Comment:
```suggestion
          ProjectUtils.logWarning(
```
`logDebug` isn't shown to the user and not being able to start the tunnel will result in missing functionality so I we should probably surface it.